src/Resources/views/form/integrated.html.twig

```diff
@@ -1,5 +1,9 @@
 <script type="text/javascript" src="https://cdn.payplug.com/js/integrated-payment/v1/index.js"></script>
 <script type="text/javascript">
+  {% set initPaymentRoute = path('payplug_sylius_integrated_payment_init', {'paymentMethodId': paymentMethod.id}) %}
+  {% if order is defined and order.getCheckoutCompletedAt is not null and order.tokenValue is not null %}
+      {% set initPaymentRoute = path('payplug_sylius_integrated_payment_init', {'paymentMethodId': paymentMethod.id, 'orderToken': order.tokenValue}) %}
```

Comment:
you could also `|merge` an object of params depending of these conditions bcoz it is the same route but with a different set of params :shrug:

Reply:
Let me know if it's better @maxperei

Reply:
yes it is @Jibbarth :1st_place_medal: